tests/fuzzer.c

```diff
@@ -1960,6 +1980,34 @@ static int basicUnitTests(U32 const seed, double compressibility)
         DISPLAYLEVEL(3, "OK \n");
     }
 
+    DISPLAYLEVEL(3, "test%3i : ZSTD_getSequences decode from sequences test : ", testNb++);
+    {
+        size_t srcSize = sizeof(U32) * 1000;
+        BYTE* src = (BYTE*)CNBuffer;
+        BYTE* decoded = (BYTE*)compressedBuffer;
+
+        ZSTD_CCtx* cctx = ZSTD_createCCtx();
+        ZSTD_Sequence* seqs = (ZSTD_Sequence*)malloc(srcSize * sizeof(ZSTD_Sequence));
+        size_t seqsSize; size_t i;
+        U32 randSeed = seed;
+
+        if (seqs == NULL) goto _output_error;
+        assert(cctx != NULL);
+
+        /* Populate src with random data */
+        for (i = 0; i < srcSize / sizeof(U32); ++i) {((U32*)CNBuffer)[i] = FUZ_rand(&randSeed);}
```

Comment:
This is likely to produce completely incompressible data, meaning 1 large litlength. We have a function to do this. You can set `srcSize == CNBuffSize`.

```
RDG_genBuffer(CNBuffer, CNBuffSize, compressibility, 0., seed);
```

Reply:
The memcmp assert fails when I'm using data generated using RDG_genBuffer(...). FUZ_decodeSequences looks fine to me so I think there is something wrong with the rep code parsing in the getSequences api. Need to double check this